config/initializers/devise.rb

```diff
@@ -12,7 +12,7 @@
   # Configure the e-mail address which will be shown in Devise::Mailer,
   # note that it will be overwritten if you use your own mailer class
   # with default "from" parameter.
-  config.mailer_sender = "no-reply@#{ENV.fetch('UNBOUNDED_DOMAIN', 'unbounded.org')}"
+  config.mailer_sender = "no-reply@#{ENV.fetch('APPLICATION_DOMAIN', 'unbounded.org')}"
```

Comment:
here again probably example.org so that mail doesn't bounce back to unbounded if a new organization uses this code and does not change this value.